src/WikiService.Data/RubricRepository.cs

```diff
@@ -112,5 +112,32 @@ public async Task<bool> DoesRubricNameExistAsync(Guid? rubricParentId, string na
     {
       return await _provider.Rubrics.AnyAsync(p => p.ParentId == rubricParentId && p.Name == nameRubric);
     }
+
+    public async Task<List<DbRubric>> GetSubRubricsAsync(GetRubricFilter filter)
```

Comment:
why are you creating a separate method? move it to the "get" method

Reply:
Done